### PR TITLE
Add OOI ERDDAP video

### LINKED ISSUE #289: Adding ERDDAP video to the Video Tutorial page
@ocefpaf  - @dpsnowden asked me to add this [OOI ERDDAP video demo](https://www.youtube.com/watch?v=tj4M9hodTG0) to our [Video Tutorial page](http://ioos.github.io/notebooks_demos/video_tutorials/).  I was hunting around the code but  couldn't find where the file where the links were to the "Other Resources" list on the left side of the Video Tutorial page.  Would you please add this video link as # 6 on the list of "Other Resources" ... or tell me what file to change  😉  .  Thanks!

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #289

### CODE REVIEW

File: webpage/video_tutorials.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ layout: single
 - [How to use the GLOS data portal](https://www.youtube.com/playlist?list=PLEK-mxRrHSZ5LkSSUyYHEy4lEXnEdNJJc)
 - [How to use the GLOS boater's tool](https://www.youtube.com/playlist?list=PLEK-mxRrHSZ6rH1qCpfpD6-rSljE8fbUC)
 
+![OOI logo]({{ site.url }}/{{ site.baseurl }}/assets/OOI_Logo.png) OOI Demonstration Videos
```
Comment (ocefpaf) [inline webpage/video_tutorials.md]: @jbosch-noaa I created a new category to add OOI videos, does that make sense?

**Comment by ocefpaf** [comment]
I need to change the OOI logo size before we merge this, it is too big...

File: webpage/video_tutorials.md (hunk 1 above)
Comment (jbosch-noaa) [reply webpage/video_tutorials.md]: Sure.  That's a good idea.  I'm sure there will be more OOI videos. 😃

**Comment by ocefpaf** [comment]
@jbosch-noaa this should be good to go now.

**Pull request merged**